### DISCUSSION TITLE
How to disable options in a checkboxlist

### DISCUSSION THREAD

**Discussion by stefanfortuin** [filamentphp/filament#2180] [Help] [answered]
I dont see a way to disable specific options in a checkboxlist.

What i am trying to do is:
A user has a role but can also have direct permissions besides the permissions in the role.
When a role gets added it will update the checkboxlist permissions according to the selected role.

The permissions that are getting selected via the role option should be disabled.

How can i disable specific checkboxlist options.

In the documentation i cannot find a way to do this.

**Comment by danharrin** [comment, marked as answer]
That's not possible with the checkbox list component at the moment, as it's not just as simple as disabling the checkbox, you also need to make sure that the user doesn't alter the HTML and toggle something they shouldn't

**Reply by stefanfortuin** [reply]
hmm okay, thanks for the reply
